CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- No changes yet.
+- Delete the ability to explicitly specify multiple files, and have the effect
```

Comment:
should be under a `### Removed` header

Reply:
It's really `Changed`, since it's remove multiple files, have single file be old dir mode.